parser/parser.go

```diff
@@ -292,29 +292,29 @@ func (p *Parser) parseMultiVariableStatement(varType *node.TypeNode, id string)
 	return mv
 }
 
-func (p *Parser) parseAssignmentStatement(left *node.DesignatorNode) *node.AssignmentStatementNode {
+func (p *Parser) parseAssignmentStatement(left node.Node) *node.AssignmentStatementNode {
 	p.nextToken() // skip '=' sign
 
 	expression := p.parseExpression()
 	p.checkAndSkipNewLines(token.NewLine)
 
 	return &node.AssignmentStatementNode{
-		AbstractNode: left.AbstractNode,
+		AbstractNode: p.newAbstractNode(),
```

Comment:
Assignment's position should be the position of designator

Reply:
Fixed https://github.com/bazo-blockchain/lazo/pull/55/commits/695f45cfe8798a3abf2696edbdb9dcaa87988c35